content/why-pulumi/delivering-cloud-native-infrastructure-as-code.md

```diff
@@ -6,6 +6,8 @@ layout: whitepaper
 
 meta_desc: In this paper, we present a consistent cloud programming model using general purpose programming languages for interacting with cloud resources.
 
+exec_summary: To a first approximation, all developers are cloud developers, all applications are cloud native, and all operations are cloud-first. Yet, there is a lack of a consistent approach to delivering cloud native applications and infrastructure. The tools and processes differ by technology generation, and even by cloud vendor, and so deny the full potential of cloud native application delivery.
```

Comment:
Just to note, I'm hoping we can retire this whitepaper in early 2020. It reflects our positioning from about a year ago -- which wasn't working quite as well as the new infrastructure as code oriented approach.